docs/source/tutorials/public-datasets.ipynb

```diff
@@ -74,11 +74,11 @@
    "id": "7",
    "metadata": {},
    "source": [
-    "For this tutorial we will limit ourselves to the `ToyDataset` due to its minimal space requirements.\n",
+    "For this tutorial we will limit ourselves to the {py:class}`~pymovements.datasets.toy_dataset.ToyDataset` due to its minimal space requirements.\n",
     "\n",
-    "Other datasets can be downloaded by simply replacing `ToyDataset` with one of the other available datasets.\n",
+    "Other datasets can be downloaded by simply replacing {py:class}`~pymovements.datasets.toy_dataset.ToyDataset` with one of the other available datasets.\n",
```

Comment:
I think in this instance just `ToyDataset` without the reference is better, as it's already linked above and it refers to the particular string here.